specification/monitor/resource-manager/Microsoft.Insights/stable/2019-03-01/actionGroups_API.json

```diff
@@ -0,0 +1,862 @@
+{
```

Comment:
In the future, for new API versions, please copy the prior API version as the first commit. That makes diffing in the PR possible. Thanks